src/psyclone/dynamo0p3.py

```diff
@@ -7131,155 +7406,189 @@ def create_halo_exchanges(self):
                     # or not
                     self._add_halo_exchange(halo_field)
 
+    @property
+    def start_expr(self):
+        '''
+        :returns: the PSyIR for the lower bound of this loop.
```

Comment:
return or returns, that is the question.

Reply:
Many happy ones, obviously.